packages/quantic/force-app/main/default/lwc/quanticCaseClassification/quanticCaseClassification.js

```diff
@@ -326,15 +328,15 @@ export default class QuanticCaseClassification extends LightningElement {
     const value = event.target.value;
     this.setFieldValue(value);
     if (this._isSuggestionsVisible && this.isMoreOptionsVisible) {
-      this.hideSuggestions();
+      this.animateHideSuggestions();
     }
   }
 
   /**
    * Hide the suggested options.
    * @returns {void}
    */
-  hideSuggestions() {
+  animateHideSuggestions() {
```

Comment:
Why change the name?

Reply:
Just because I needed to create a property which contains a boolean value to indicate if the suggestions must be hidden or not, the appropriate name for this new property was `hideSuggestions`, so I updated the name of this method to differentiate  them, and `animateHideSuggestions` is the appropriate name of this method since its the method responsible of doing the animation to hide the suggestions.